ncm-network/src/main/pan/components/network/core-schema.pan

```diff
@@ -18,6 +18,7 @@ type structure_interface_alias = {
     "ip"      ? type_ip
     "netmask" : type_ip
     "broadcast" ? type_ip
+    "fqdn"    ? string
```

Comment:
that makes sense, but how is this used? e.g. is there a component that uses this? (i'm just curious)